<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QMainWindow" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>580</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1060</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1060</width>
    <height>580</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Whiteboard</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>422</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Width:</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Colour:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>Black</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>White</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Red</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Blue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Green</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Magenta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Yellow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Custom</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Background Colour:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>White</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Black</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cyan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Magenta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Yellow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Custom</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_send">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>512</width>
      <height>480</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_receive">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>512</width>
      <height>480</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>456</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Receive</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>521</x>
      <y>10</y>
      <width>20</width>
      <height>530</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="xMousePos">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="plainText">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="yMousePos">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="plainText">
     <string>0</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cBoxLoop">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>10</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Local loopback</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1060</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClear"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
